src/util.ts

```diff
@@ -1175,6 +1173,23 @@ export class Util {
             range: attr.range
         } as SGAttribute;
     }
+
+    /**
+     * Given a pkgPath, remove the `pkg:/` portion of the path.
+     */
+    public removePkgProtocol(pkgPath: string) {
+        if (pkgPath.startsWith('pkg:/')) {
```

Comment:
Maybe add `libpkg:`?

Reply:
Handled in 50a155549b9b4a9fbacbef8ecc769109d6abc66e